Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:7-onbuild
+
+# Install node
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev libkrb5-dev
+
+# Install yarn
+RUN set -xe \
```

Comment:
Yarn doesn't actually buy us anything for the docker case because all dependencies are pegged in package.json. This would simplify the install process here a bit